backend/app/auth/routes.py

```
@@ -14,32 +15,23 @@
-)
+from fastapi import APIRouter, Depends, HTTPException, Request, status
+from fastapi.security import OAuth2PasswordRequestForm
 
 router = APIRouter(prefix="/auth", tags=["Authentication"])
 
 
-@router.post(
-    "/token", response_model=TokenResponse, include_in_schema=False
-)  # include_in_schema=False to hide from docs if desired, or True to show
+@router.post("/token", response_model=TokenResponse, include_in_schema=False)
 async def login_for_access_token(form_data: OAuth2PasswordRequestForm = Depends()):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)